src/fundamental/tpm-pcr.h

```diff
@@ -31,6 +31,7 @@ typedef enum UnifiedSection {
         UNIFIED_SECTION_DTB,
         UNIFIED_SECTION_PCRSIG,
         UNIFIED_SECTION_PCRPKEY,
+        UNIFIED_SECTION_UNAME,
```

Comment:
given then pcr sig covers the uname too now I think we should place the uname enum value before the pcrsig enum value here. it's conceptually a bit weird if an early section "covers" a later section I think. and given that the order is not API we can just makes this straight-forward and place the covered sections first.